.travis.yml

```diff
@@ -144,7 +147,4 @@ notifications:
     skip_join: true
   email:
     - pytest-commit@python.org
-cache:
-    directories:
-        - $HOME/.cache/pip
-        - $HOME/.cache/pre-commit
+cache: false
```

Comment:
this is the default iirc

Reply:
Fine if we want to keep it, but I suggest to add a comment like the one you put on the description of this PR for reference. 👍